e2e-tests/portal/cypress/fixtures/users.js

```diff
@@ -1,33 +1,26 @@
 const MOCK_USERS = require('../../../../utils/mock-data-loader/portal/users');
 const MOCK_TFM_USERS = require('../../../../utils/mock-data-loader/tfm/mocks/users');
 
-const BANK1_MAKER1 = MOCK_USERS.find((user) =>
-  (user.roles.includes('maker') && user.username === 'BANK1_MAKER1'));
+const BANK1_MAKER1 = MOCK_USERS.find((user) => user.roles.includes('maker') && user.username === 'BANK1_MAKER1');
```

Comment:
Please use constants for `role` and `username`.

Reply:
updated `role`, i've left `username` as we're not currently doing this anywhere else and it would be good to do it all in one go to avoid fragmentation.

Reply:
Sounds sensible, cannot see `role` `maker` as constant, not sure if it has been pushed?